src/Compilers/CSharp/Portable/Symbols/Source/SourceDelegateMethodSymbol.cs

```diff
@@ -27,7 +27,7 @@ internal abstract class SourceDelegateMethodSymbol : SourceMemberMethodSymbol
             : base(delegateType, syntax.GetReference(), location: syntax.Identifier.GetLocation(), isIterator: false)
         {
             _returnType = returnType;
-            this.MakeFlags(methodKind, declarationModifiers, _returnType.IsVoidType(), isExtensionMethod: false);
+            this.MakeFlags(methodKind, declarationModifiers, _returnType.IsVoidType(), isExtensionMethod: false, isNullableAnalysisEnabled: false);
```

Comment:
Delegates do allow for default parameter values. Will disabling nullable analysis at this level prevent the analysis of delegate parameter default values?

Reply:
> Will disabling nullable analysis at this level prevent the analysis of delegate parameter default values?

No. The decision on whether to analyze parameter default values is handled separately, in `SourceComplexParameterSymbol.DefaultSyntaxValue`.

---
In reply to: [551610333](https://github.com/dotnet/roslyn/pull/49953#discussion_r551610333) [](ancestors = 551610333)